# list of the obstacles
ObstacleMsg[] obstacles